README.md

```diff
@@ -1,13 +1,80 @@
 # REDCap User Profile
-This is a REDCap external module that provides user profiles management - in other words, it gives the administrators the ability to create (and manage) additional user account information such as address, country of birth, job position, etc.
+REDCap User Profile is an external module that extends user accounts information according to your needs - e.g. address, country of birth, job position, etc. This module provides:
+- An easy way to manage user profiles
+- An API to assist developers in accessing user profiles information
+
+## How does it work?
+This module assigns to a REDCap project the role of representing and storing user profile entities as data entry records.
 
 ## Prerequisites
 - [REDCap Modules](https://github.com/vanderbilt/redcap-external-modules)
 
-
 ## Installation
 - Clone this repo into to `<redcap-root>/modules/redcap_user_profile_v1.0`.
-- Go to **Control Center > Manage External Modules** and enable User Profile.
+- Go to **Control Center > Manage External Modules** and enable User Profile for all modules.
 
 ## Configuration
-TODO.
+
+### Making sure Table-based authentication is enabled
+Make sure that Table-based authentication is enabled in your REDCap, since it is required to manage user accounts.
```

Comment:
Saying "table-based authentication" and in the heading above is too specific. How about this:

> If you are working in a test instance, you must turn on some form of authentication as it builds upon account management features.  Table-based authentication will work fine.

Reply:
I like that!